engine/runtime-integration-tests/src/test/java/org/enso/interpreter/test/interop/AtomInteropTest.java

```diff
@@ -481,6 +484,8 @@ public void invokeLazyField_DoesNotCauseStackOverflow() {
         type Generator
             Value n ~next
 
+            get_next self = self.next
```

Comment:
What is the purpose of this change?

Reply:
Debris removed in 14632b2a7360d4b21b7484d86007c734d3d31da8. Originally I started by modifying this test and only later I copied it. Accidentally some debris has been left.